FAQ's.md

```diff
@@ -0,0 +1,30 @@
+**Q: What is ADSdroid ?**
+
+A: It is the unofficial AllDataSheet Android Application. It gives her users the privilege of searching and downloading electronics tools in pdf format, for easier and faster access 
```

Comment:
not `the unofficial` but rather `an unofficial`

Reply:
also, `gives her users the privilege of searching and downloading` should be `makes it possible for users to search and download`

Reply:
PDF is an acronym, it should be all caps

Reply:
`electronics tools` is a strange term, `datasheets for electronic components` would be better